src/assets/js/app.js

```diff
@@ -9,7 +9,7 @@ import Foundation from 'foundation-sites';
 //import './lib/foundation-explicit-pieces';
 
 import 'tablesaw/dist/tablesaw.jquery';
-import libs from './lib/dependancies';
+import libs from './lib/dependencies';
```

Comment:
Actually, the file is named [`dependancies.js`](https://github.com/zurb/foundation-zurb-template/blob/master/src/assets/js/lib/dependancies.js). Could you rename the file too ?

Reply:
It's already done in the pull request.

 src/assets/js/lib/dependancies.js → src/assets/js/lib/dependencies.js

Reply:
Oh, sorry, didn't seen that. LGTM then!